service-deployment/bootstrap/prometheus/prometheus-configmap.yaml.template

```diff
@@ -30,4 +30,11 @@ data:
         {% for host in machinelist %}
         - {{ machinelist[host]['ip']}}:{{ clusterinfo['prometheusinfo']['node_exporter_port'] }}
         {% endfor %}
+    - job_name: "cadvisor"
+      scrape_interval: "15s"
+      static_configs:
+      - targets: 
+        {% for host in machinelist %}
+        - {{ machinelist[host]['ip']}}:{{ clusterinfo['prometheusinfo']['cadvisor_port'] }}
```

Comment:
Aggregate all host infomation through ca in one node?

Reply:
This will aggregate all node's cadvisor's metrics, as cadvisor expose prometheus format metrics through this port. current version use Kube default cadvisor, if deploy by ourself will add node label.